R/init.R

```diff
@@ -45,14 +42,9 @@
 #'
 #' @examples
 #' app <- init(
-#'   data = teal_data(
-#'     new_iris = transform(iris, id = seq_len(nrow(iris))),
-#'     new_mtcars = transform(mtcars, id = seq_len(nrow(mtcars))),
-#'     code = "
-#'       new_iris <- transform(iris, id = seq_len(nrow(iris)))
-#'       new_mtcars <- transform(mtcars, id = seq_len(nrow(mtcars)))
-#'     "
-#'   ),
+#'   data = teal_data() |>
```

Comment:
`teal` depends on R >=4.0. What would you prefer?
- bump R >= 4.1 or 
- don't use `|>`

Reply:
Right now I will change it not to use `|>` not to explode this PR. However I think that we can do vbump. This will play nicely with the recent change of not re-exporting `%>%` which was a good call.